sopel/plugins/handlers.py

```diff
@@ -330,3 +330,51 @@ def reload(self):
         module might not be available through ``sys.path``.
         """
         self._module = self._load()
+
+
+class EntrypointPluginHandler(PyModulePlugin):
```

Comment:
Any particular reason this class is named with the `Handler` suffix when the others are just `TypeOfPlugin`, without mentioning `Handler` anywhere?

This goes along with another line comment I haven't added yet in another file: Why isn't it `EntryPoint` in CamelCase? All over the Python docs (and the ones you added here), they're referred to as "entry points"—two words—so why does the code itself refer to `entrypoint(s)`?

Based on those two things, I think this class should actually be called `EntryPointPlugin`.

Reply:
Sometimes, I don't make the right decision. That's why review exists and I'm glad they do!

Reply:
```suggestion
class EntryPointPlugin(PyModulePlugin):
```